src/core/backend.h

```diff
@@ -102,6 +102,9 @@ class InferenceBackend {
       const uint32_t runner_cnt, Scheduler::StandardInitFunc OnInit,
       Scheduler::StandardRunFunc OnRun);
 
+  // Get the raw pointer to the scheduler of the servable.
+  Scheduler* GetRawScheduler() { return scheduler_.get(); }
```

Comment:
Just "Scheduler". I know there are some Get prefixes on other functions but I plan to fix those.